src/qcodes/dataset/sqlite/connection.py

```diff
@@ -21,7 +21,7 @@
 log = logging.getLogger(__name__)
 
 
-class ConnectionPlus(wrapt.ObjectProxy):
+class ConnectionPlus(wrapt.ObjectProxy):  # pyright: ignore[reportUntypedBaseClass]
```

Comment:
what's the reason for untyped base class error? wrapt is not pyted or doesn't have stubs? :(